hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/QueryInfo.java

```diff
@@ -33,6 +37,7 @@ public class QueryInfo {
   private final String previousInstant;
   private final String startInstant;
   private final String endInstant;
+  private final String predicateFilter;
```

Comment:
can we add some java docs on each entity here. We keep adding more and more. some of them might be obvious.but good to have some docs.